src/siembol/resources/application.properties

```diff
@@ -0,0 +1,67 @@
+management.endpoint.health.show-details=always
+management.endpoints.web.exposure.include=info,prometheus,health
+management.endpoints.web.path-mapping.prometheus=metrics
+management.endpoints.web.base-path=
+management.endpoint.health.status.http-mapping.UP=200
+management.endpoint.health.status.http-mapping.UNKNOWN=200
+management.endpoint.health.status.http-mapping.DOWN=503
+management.endpoint.health.status.http-mapping.OUT_OF_SERVICE=503
+
+server.port=8081
+# server.max-http-header-size=5242880
+
+# info.env.java.vendor=${java.specification.vendor}
+# info.env.java.vm-name=${java.vm.name}
+# info.env.java.runtime-version=${java.runtime.version}
+# info.env.java.app.description=Siembol config editor
+
+# logging.level.org.apache.http=INFO
+# logging.level.org.apache.http.wire=INFO
+# logging.level.org.springframework.web.filter.CommonsRequestLoggingFilter=INFO
+# logging.level.org.springframework.web=INFO
+# logging.level.org.springframework.security=INFO
+# logging.level.org.springframework.web.servlet.mvc.method.annotation.RequestResponseBodyMethodProcessor=INFO
+# logging.level.org.springframework.web.FilterChainProxy=INFO
+
+config-editor-auth.type=disabled
+# config-editor-auth.oauth2.excluded-url-patterns=/info,/health,/health/liveness,/health/readiness,/metrics,/v3/api-docs/**,/swagger-ui/**,/swagger-ui.html
+# config-editor-auth.oauth2.audience=siembol
+# config-editor-auth.oauth2.tokenUrl=https://your-auth-server/connect/token
+# config-editor-auth.oauth2.authorizationUrl=https://your-auth-server/connect/authorize
+# config-editor-auth.oauth2.issuerUrl=https://your-auth-server
+# config-editor-auth.oauth2.jwkSetUrl=https://your-auth-server/.well-known/openid-configuration/jwks
+# config-editor-auth.oauth2.jwsAlgorithm=RS256
+# config-editor-auth.oauth2.jwtType=at+jwt
+# config-editor-auth.oauth2.scopes=siembol,openid
+
+# springdoc.pathsToMatch=/health,/info,/metrics,/prometheus,/api/**,/user
+# springdoc.show-actuator=true
+
+# springdoc.swagger-ui.oauth.clientId=siembol.swagger
+# springdoc.swagger-ui.oauth.appName=siembol
+# springdoc.swagger-ui.oauth.clientSecret=siembol.swagger
+# springdoc.swagger-ui.oauth.usePkceWithAuthorizationCodeGrant=true
+
+config-editor.test-cases-ui-config-file-name=ui-config/test-cases-layout-config.json
+
+config-editor.synchronisation=ALL
+config-editor.storm-topologies-zookeeper.zk-path=/siembol/synchronise
+config-editor.storm-topologies-zookeeper.zk-url=siembol-zookeeper:2181
+
+config-editor.services.alert.type=alert
+config-editor.services.alert.config-store.git-user-name=stackedsax
+config-editor.services.alert.config-store.git-password=${GIT_PSWD}
+config-editor.services.alert.config-store.github-url=https://github.com
```

Comment:
we can use environment variables for github url and github repository name

Reply:
I think that's the right thing to do.  I'm assuming that I would just add something like this to the correct deployment templates:

```
env:
  - name: GITHUB_URL
     value: https://github.com
```

And then the variable would be available in the container.

At that point, we could have the `application.properties` read:
```
config-editor.services.alert.config-store.github-url=${GITHUB_URL}
```
But could we take this line out of `application.properties` for the quickstart entirely?  I'm all for having as little in the initial `application.properties` as possible and making it feel less intimidating.  We can then document all the things you can override by setting values in `application.properties` for the more advanced folks.

If we take this line out, where does `${GITHUB_URL}` need to show up in the container?

Reply:
Yes this we can discuss in more datail. But the idea can be use in testing deployment properties from docker image and we will use an "env variable trick" to initiate custom values like git user, git repo,..

Reply:
As we discussed, I'm going to leave this as it is for the moment and we can tackle this in a separate issue.